### PR TITLE
update inject, arquillian-weld-embedded, cdi-tck

### PR DESCRIPTION
**Pull request by gurunrao**
Signed-off-by: Gurunandan <gurunandan.rao@oracle.com>
DI porting TCK upgrade to use inject 3.0.0 and glassfish 6.
CI run - https://ci.eclipse.org/jakartaee-tck/job/ditck-porting-grao/job/master/lastBuild/

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -48,19 +48,19 @@ spec:
   }
   parameters {
     string(name: 'GF_BUNDLE_URL', 
-           defaultValue: 'https://download.eclipse.org/ee4j/jakartaee-tck/8.0.1/nightly/glassfish.zip', 
+           defaultValue: 'https://ci.eclipse.org/jakartaee-tck/job/build-glassfish/lastSuccessfulBuild/artifact/appserver/distributions/glassfish/target/glassfish.zip', 
            description: 'URL required for downloading GlassFish Full/Web profile bundle' )
     string(name: 'JAVAX_INJECT_TCK_URL',
```
Comment (alwin-joseph) [inline Jenkinsfile]: Minor comment : Maybe better to change this parameter name to avoid javax.

File: build.xml [hunk 2]
```diff
@@ -45,7 +45,7 @@
                         <include name="**/*.jar" /> <exclude name="**/javax.inject*.jar"/>
 		</fileset>
 		<fileset dir="${tck.home}">
-                        <include name="jakarta.inject-tck-1.0.jar" />
+                        <include name="jakarta.inject-tck-2.0.0.RC3.jar" />
```
Comment (alwin-joseph) [inline build.xml]: Not sure if it is easy to parameterise this value. Will it not be required to change to final jar name later ?

**Review by alwin-joseph — APPROVED** [review]
Left some minor comments. Feel free to ignore.

File: Jenkinsfile (hunk 1 above)
Comment (gurunrao) [reply Jenkinsfile]: done.

File: build.xml (hunk 2 above)
Comment (gurunrao) [reply build.xml]: removed version

**Pull request merged**